agent/stats/common_test.go

```diff
@@ -170,6 +179,65 @@ func validateMetricsMetadata(metadata *ecstcs.MetricsMetadata) error {
 	return nil
 }
 
+func validateHealthMetricsMetadata(metadata *ecstcs.HealthMetadata) error {
+	if metadata == nil {
+		return fmt.Errorf("metadata is nil")
+	}
+
+	if aws.StringValue(metadata.Cluster) != defaultCluster {
+		return fmt.Errorf("expected cluster in metadata to be: %s, got %s",
+			defaultCluster, aws.StringValue(metadata.Cluster))
+	}
+
+	if aws.StringValue(metadata.ContainerInstance) != defaultContainerInstance {
+		return fmt.Errorf("expected container instance in metadata to be %s, got %s",
+			defaultContainerInstance, aws.StringValue(metadata.ContainerInstance))
+	}
+	if len(aws.StringValue(metadata.MessageId)) == 0 {
+		return fmt.Errorf("empty MessageId")
+	}
+
+	return nil
+}
+
+func validateContainerHealthMetrics(metrics []*ecstcs.ContainerHealth, expected int) error {
+	if len(metrics) != expected {
```

Comment:
nit: can use `require.Len` here

Reply:
The `testing.T` isn't passed in this method, though we probably should, let's take it out of this PR, I will refactor this part with another pr.